src/components/SideBar/SideBar.jsx

```diff
@@ -186,12 +186,10 @@ class SideBar extends PureComponent {
    * @param keyPath - is the path from root to current node
    */
   onNodeMouseClick(event, tree, node) {
```

Comment:
Can't we leverage the AppUtils.getHomePath function here?

Reply:
You are correct, updated with these changes.